resources/assets/components/utilities/ModalRoute/ModalRoute.js

```diff
@@ -53,12 +53,12 @@ class ModalRoute extends React.Component {
         />
         {isModal(location) ? (
           <StaticRouter location={withoutModal(location)} context={{}}>
-            <div>{children}</div>
+            {children}
           </StaticRouter>
         ) : (
-          <div>{children}</div>
+          <>{children}</>
```

Comment:
Does this need a fragment around it? I believe this could just be:

```suggestion
          children
```